charts/values.yaml.changelog.md

```diff
@@ -5,6 +5,9 @@ Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
 ----
+## Upgrading to Renku 0.7.0
+* EDIT - *postgresql.persistence.existingClaim*: Renku `0.7.0` upgrades the postgres chart dependency, which requires modification of the postgres data volume of existing deployments. See [this helper job manifest](https://github.com/SwissDataScienceCenter/renku/blob/master/charts/utils/migrate-pg-to-bitnami.yaml).
```

Comment:
I think we need a stronger message here - this is a breaking change

Reply:
We've had those before without explicitly mentioning, but I agree it would be good practice, especially while we're pre 1.0 where semantic versioning is not as unambiguous as after 1.0. Adapted.